### DISCUSSION TITLE
Separate repository

### DISCUSSION THREAD

**Discussion by AzeemHadzrie** [iptv-org/community#45] [Ideas]
Shall anyone do a separate repository that has VOD movies, shows, etc. Anyone else might wan't it.

**Comment by BellezaEmporium** [comment]
> Shall anyone do a separate repository that has VOD movies, shows, etc. Anyone else might wan't it.

It's a bit more borderline... unless you only reference Public Domain movies and shows...

**Comment by AzeemHadzrie** [comment]
For example, from Paramount+, Hulu, and more streaming services (that have VOD movies/TV series)

**Reply by benevenstanciano** [reply]
Definitely a no-no. All of those shows would be hosted illegally.

**Reply by AzeemHadzrie** [reply]
Alright then.
